### PR TITLE
Model updates 2022-09-01

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/geopol.py [hunk 1]
```diff
@@ -40,13 +42,42 @@ def getModelDefs(self):
...
+                        ('name', ('geo:name', {}), {
+                            'doc': 'The name of the country.'}),
+                        ('names', ('array', {'type': 'geo:name', 'uniq': True, 'sorted': True}), {
+                            'doc': 'An array of alternate or localized names for the country.'}),
+                        ('place', ('geo:place', {}), {
+                            'doc': 'A geo:place node representing the geospatial properties of the country.'}),
+                        ('founded', ('time', {}), {
+                            'doc': 'The date that the country was founded.'}),
+                        ('dissolved', ('time', {}), {
+                            'doc': 'The date that the country was disolved.'}),
```
Comment (Cisphyx) [inline synapse/models/geopol.py:62]: typo 'disolved'

File: synapse/models/geospace.py [hunk 2]
```diff
@@ -301,6 +381,9 @@ def getModelDefs(self):
                     ('geo:dist', 'synapse.models.geospace.Dist', {}, {
                         'doc': 'A geographic distance (base unit is mm).', 'ex': '10 km'
                     }),
+                    ('geo:area', 'synapse.models.geospace.Area', {}, {
+                        'doc': 'A geographic area (base unit is square mm).', 'ex': '10 sq/km'
```
Comment (Cisphyx) [inline synapse/models/geospace.py:385]: 'sq/km' isn't in the list of area units

Comment (infime) [inline synapse/models/geospace.py:385]: should be `sq.km`?

File: synapse/models/inet.py [hunk 3]
```diff
@@ -2394,7 +2396,23 @@ def getModelDefs(self):
...
+                            'doc': 'The account that is a member of the group or channel.'
+                        }),
+                        ('group', ('inet:web:group', {}), {
+                            'doc': 'The group that the account is a member of.'
+                        }),
+                        ('channel', ('inet:web:channel', {}), {
+                            'doc': 'The channel that the account is a member of.'
+                        }),
+                        ('added', ('time', {}), {
+                            'doc': 'The date / time the account as added to the group or channel.'
```
Comment (Cisphyx) [inline synapse/models/inet.py:2410]: missing 'w' in 'account as added'

File: synapse/models/geospace.py [hunk 4]
```diff
@@ -197,16 +232,15 @@ def _normPyStr(self, text):
         try:
             valu, off = s_grammar.parse_float(text, 0)
         except Exception:
-            raise s_exc.BadTypeValu(valu=text, name=self.name,
-                                    mesg='Dist requires a valid float and dist '
-                                         'unit, no valid float found') from None
+            mesg = 'Distance requires a valid number and unit. No valid number found.'
```
Comment (vEpiphyte) [inline synapse/models/geospace.py:235]: We should add `{valu}` here

File: synapse/models/geospace.py [hunk 5]
```diff
@@ -236,6 +270,52 @@ def repr(self, norm):
...
+        self.setNormFunc(str, self._normPyStr)
+
+    def _normPyInt(self, valu):
+        return valu, {}
+
+    def _normPyStr(self, text):
+        try:
+            valu, off = s_grammar.parse_float(text, 0)
+        except Exception:
+            mesg = 'Area requires a valid number and unit, no valid number found.'
```
Comment (vEpiphyte) [inline synapse/models/geospace.py:288]: We should add {valu} here

**Review by vEpiphyte — APPROVED** [review]

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**